PyInstaller/utils/hooks/__init__.py

```diff
@@ -365,7 +365,7 @@ def get_pywin32_module_file_attribute(module_name):
     return get_module_attribute(module_name, '__file__')
 
 
-def check_requirement(requirement: str):
+def check_requirement(requirement: str, match_prereleases=True):
```

Comment:
Are we ever likely to want to set `match_prereleases` to false?

Reply:
Considering it affects matching of higher-level pre-releases, I suppose the answer is no.

Reply:
And if we ever do, we can add the parameter at that point.